lib/get-package-path.js

```diff
@@ -1,16 +1,17 @@
 const path = require('path');
 const loadJsonFile = require('load-json-file');
-const pkgDir = require('pkg-dir');
+const pkgUp = require('pkg-up');
 
 const DEFAULT_PACKAGE = {xo: false};
 
 // (base: string) => pkgPath: Promise<string>
 async function findPackage(base) {
-	const pkgPath = path.join(base, 'package.json');
+	const pkgPath = await pkgUp({cwd: base});
 	const pkg = await loadPkg(pkgPath);
 
-	if (pkg.xo === false && base !== null) {
-		return findPackage(path.join(base, '..'));
```

Comment:
`path.join(base, '..')` was never `null`, so if a `package.json` wasn't found, `findPackage('/')` would be called repeatedly